cristin-person/src/test/java/no/unit/nva/cristin/person/employment/delete/DeletePersonEmploymentHandlerTest.java

```diff
@@ -0,0 +1,102 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class DeletePersonEmploymentHandlerTest {
+
+    private static final String EMPTY_JSON = "{}";
+    private static final String INVALID_PERSON_ID = "abc";
+    private static final String INVALID_EMPLOYMENT_ID = "def";
+    private static final String VALID_PERSON_ID = "123456789";
+    private static final String VALID_EMPLOYMENT_ID = "987654321";
+    private final HttpClient httpClientMock = mock(HttpClient.class);
+    private final Environment environment = new Environment();
+    private DeletePersonEmploymentClient apiClient;
+    private Context context;
+    private ByteArrayOutputStream output;
+    private DeletePersonEmploymentHandler handler;
+
+    @BeforeEach
+    void setUp() throws IOException, InterruptedException {
+        when(httpClientMock.<String>send(any(), any())).thenReturn(new HttpResponseFaker(EMPTY_JSON, 200));
+        apiClient = new DeletePersonEmploymentClient(httpClientMock);
+        context = mock(Context.class);
+        output = new ByteArrayOutputStream();
+        handler = new DeletePersonEmploymentHandler(apiClient, environment);
+    }
+
+
+    @Test
+    void shouldThrowForbiddenExceptionWhenClientIsNotAuthenticated() throws IOException {
+        handler.handleRequest(getInputStreamRequestWithoutAuthentication(), output, context);
+        GatewayResponse<Void> gatewayResponse = GatewayResponse.fromOutputStream(output);
+        assertEquals(HttpURLConnection.HTTP_FORBIDDEN, gatewayResponse.getStatusCode());
+        assertEquals(APPLICATION_PROBLEM_JSON.toString(), gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+    }
+
+    @Test
+    void shouldThrowBadRequestExceptionWhenInvalidPersonId() throws IOException {
+        handler.handleRequest(getInputStreamRequestWithInvalidPersonId(), output, context);
+        GatewayResponse<Void> gatewayResponse = GatewayResponse.fromOutputStream(output);
+        assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, gatewayResponse.getStatusCode());
+        assertEquals(APPLICATION_PROBLEM_JSON.toString(), gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+    }
+
+    @Test
+    void shouldThrowBadRequestExceptionWhenInvalidEmploymentId() throws IOException {
+        handler.handleRequest(getInputStreamRequestWithInvalidEmploymentId(), output, context);
+        GatewayResponse<Void> gatewayResponse = GatewayResponse.fromOutputStream(output);
+        assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, gatewayResponse.getStatusCode());
+        assertEquals(APPLICATION_PROBLEM_JSON.toString(), gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+    }
+
+
+    private InputStream getInputStreamRequestWithoutAuthentication() throws JsonProcessingException {
+        return new HandlerRequestBuilder<CristinPersonEmployment>(OBJECT_MAPPER)
```

Comment:
You dont need CristinPersonEmployment here and the other methods but maybe it does not matter